src/Nuget.PackageIndex.VisualStudio/ProjectKFilter.cs

```diff
@@ -43,8 +86,114 @@ public bool IsProjectSupported(string filePath)
                     // TODO Package manager console?
                     Debug.Write(string.Format("{0} \r\n {1}", e.Message, e.StackTrace));
```

Comment:
Opened issue https://github.com/NuGet/NuGet.PackageIndex/issues/9  to track this